src/app/modules/notification/components/notifications/notifications.component.html

```diff
@@ -0,0 +1,13 @@
+<div *ngIf="notifications" id="notifications" class="notifications">
+  <div *ngFor="let n of notifications"
+    id="{{ n.id }}"
+    class="notifications__item notifications__item--{{ n.type }}"
+    [ngClass]="{'dismissible': n.options.isDismissible}">
```

Comment:
`dismissable` no ?

Reply:
You're right ! [Dismissable vs Dismissible - What's the difference?](https://wikidiff.com/dismissable/dismissible)